src/components/Footer/FooterView.js

```diff
@@ -2,21 +2,40 @@ import React, { useEffect, useState } from 'react';
 import { Footer } from 'bento-components';
 import FooterData from '../../bento/globalFooterData';
 import env from '../../utils/env';
-import CustomThemeProvider from './FooterThemConfig';
+import CustomThemeProvider from './FooterThemeConfig';
 
 const FILE_SERVICE_API = env.REACT_APP_FILE_SERVICE_API;
 
+const fetchVersion = (url) => fetch(`${url}version`)
+  .then((resp) => resp.text())
+  .then((text) => {
+    const json = JSON.parse(text);
+    return json.version;
+  })
+  .catch(() => '0.0.0');
+
 const ICDCFooter = () => {
   const [footerUpdatedData, setFooterUpdatedData] = useState(FooterData);
 
   useEffect(() => {
     const getSystems = async () => {
-      const response = await fetch(
-        `${FILE_SERVICE_API}version`,
-      ).then((resp) => (resp))
-        .catch(() => ({ version: '' }));
-      const data = response.json();
-      setFooterUpdatedData({ ...FooterData, ...{ FileServiceVersion: data.version || '' } });
+      const { href, hash } = window.location;
+      const hashIndex = href.indexOf(hash) || href.length;
+      const hashlessUrl = href.substring(0, hashIndex);
+      const [BEversion, FileServiceVersion] = await Promise.all([
```

Comment:
@amattu2 What if one of the Promises is rejected due to either of the services not running? will we be able to see other version numbers?